### DISCUSSION TITLE
🚀 v1.0.0

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1918] [Releases]
## 🚀 Speedtest Tracker Release v1.0.0

I'm thrilled to announce the release of **Speedtest Tracker v1.0.0**! This milestone has been a long time coming (824 days, not that I'm counting), and I'm wicked excited to _finally_ say "I think this is a good first version".

I also want to take a moment to say a special thanks to those who contributed:

- [svenvg93](https://github.com/svenvg93)
- [adamus1red](https://github.com/adamus1red)
- [UnAfraid](https://github.com/UnAfraid)
- [jonezy35](https://github.com/jonezy35)
- [wdog](https://github.com/wdog)
- [nandi95](https://github.com/nandi95)
- [garanda21](https://github.com/garanda21)
- [jocmp](https://github.com/jocmp)

To see more contributors, visit the [contributors page](https://github.com/alexjustesen/speedtest-tracker/graphs/contributors).

### Current Features

- **Automated Tests**: Schedule regular speed tests to monitor your internet connection's performance over time.
- **Detailed Metrics**: Capture download and upload speeds, ping, packet loss and more.
- **Historical Data**: View historical data and trends to identify patterns and issues with your internet connection.
- **Notifications**: Receive notifications when your internet performance drops below a certain threshold.

### Planned Features

I doubt this project will ever be "done" so here's a sneak peak for what I have planned for after the New Year.

- **Multiple Speedtest Schedules**: Support for multiple speedtest schedules and make them configurable in the UI.
- **Ping Tests**: Addition of ping tests to measure network latency.
- **Additional Speedtest Tools**: Integration with additional testing tools like iPerf, LibreSpeed and OpenSpeedTest.
- **API**: Introduction of an API for better integration with other applications and services.
- **Revamped Dashboard**: An improved and more user-friendly dashboard for better data visualization and user experience.

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v1.0.0'>🚀 v1.0.0</a>.</em>

**Comment by tomerh2001** [comment]
Hype 🥇 🚀

**Comment by luckman212** [comment]
Congratulations on the 1.0 milestone! 👍 Incredibly useful app! Running smoothly here. I have to update my docker configs to take advantage of some of the recent feature additions like server exclusions.

**Comment by agneevX** [comment]
Is there anything new in this release at all or is this simply a version bump?

**Reply by alexjustesen** [reply]
https://github.com/alexjustesen/speedtest-tracker/compare/v0.26.0...v1.0.0
